iota web go ci failed

https://github.com/dylibso/observe-sdk/commit/7e03891652192ab88d419b18b1b76fa9da18dffc didn't pass CI.

In case NF is inaccessible, I reproduced the build log below.

```
2024-08-15T14:41:10.44213615Z stdout F [2024-08-15T14:41:10Z INFO ] Process terminated with exit code 255
2024-08-15T14:41:10.43276946Z stderr F [error] BuildService - Build failed: Could not build the image.
2024-08-15T14:41:07.411247771Z stdout F error building image: error building stage: failed to execute command: waiting for process to exit: exit status 1
2024-08-15T14:41:07.407619218Z stdout F /go/pkg/mod/github.com/tetratelabs/wazero@v1.8.0/internal/engine/wazevo/ssa/basic_block_sort.go:4:2: package slices is not in GOROOT (/usr/local/go/src/slices)
2024-08-15T14:41:07.181049661Z stdout F [INFO] Running: [/bin/sh -c go build -o /iota]
2024-08-15T14:41:07.181044682Z stdout F [INFO] Args: [-c go build -o /iota]
2024-08-15T14:41:07.181024504Z stdout F [INFO] Cmd: /bin/sh
2024-08-15T14:41:05.6513083Z stdout F [INFO] RUN go build -o /iota
2024-08-15T14:41:00.794926057Z stdout F go: downloading google.golang.org/protobuf v1.34.2
2024-08-15T14:41:00.758088567Z stdout F go: downloading go.opentelemetry.io/proto/otlp v1.3.1
2024-08-15T14:41:00.758072687Z stdout F go: downloading github.com/ianlancetaylor/demangle v0.0.0-20240805132620-81f5be970eca
2024-08-15T14:41:00.756173616Z stdout F go: downloading github.com/tetratelabs/wazero v1.8.0
2024-08-15T14:40:57.766488015Z stdout F go: downloading golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
2024-08-15T14:40:57.526051886Z stdout F go: downloading github.com/google/go-cmp v0.5.5
2024-08-15T14:40:57.427793404Z stdout F go: downloading gopkg.in/yaml.v3 v3.0.1
2024-08-15T14:40:57.415648991Z stdout F go: downloading github.com/pmezard/go-difflib v1.0.0
2024-08-15T14:40:57.394162954Z stdout F go: downloading github.com/davecgh/go-spew v1.1.1
2024-08-15T14:40:57.158247284Z stdout F go: downloading github.com/stretchr/testify v1.8.0
2024-08-15T14:40:57.070735227Z stdout F go: downloading google.golang.org/protobuf v1.31.0
2024-08-15T14:40:56.941550809Z stdout F go: downloading go.opentelemetry.io/proto/otlp v1.0.0
2024-08-15T14:40:56.624888463Z stdout F go: downloading github.com/tetratelabs/wabin v0.0.0-20230304001439-f6f874872834
2024-08-15T14:40:56.623882021Z stdout F go: downloading github.com/ianlancetaylor/demangle v0.0.0-20230524184225-eabc099b10ab
2024-08-15T14:40:56.506530571Z stdout F go: downloading github.com/tetratelabs/wazero v1.4.0
2024-08-15T14:40:56.491581194Z stdout F [INFO] Running: [/bin/sh -c go mod tidy]
2024-08-15T14:40:56.491575954Z stdout F [INFO] Args: [-c go mod tidy]
2024-08-15T14:40:56.49155772Z stdout F [INFO] Cmd: /bin/sh
2024-08-15T14:40:55.679529756Z stdout F [INFO] RUN go mod tidy
2024-08-15T14:40:55.679525518Z stdout F [INFO] Changed working directory to /app/demo-iota/go/
2024-08-15T14:40:55.679520789Z stdout F [INFO] Cmd: workdir
2024-08-15T14:40:55.679506733Z stdout F [INFO] WORKDIR /app/demo-iota/go/
2024-08-15T14:40:55.677333397Z stdout F [INFO] ADD /demo-iota/go /app/demo-iota/go
2024-08-15T14:40:55.677320252Z stdout F [INFO] Using files from context: [/etc/.ci-agent/workspace/demo-iota/go]
2024-08-15T14:40:55.662401863Z stdout F [INFO] ADD /go /app/go
2024-08-15T14:40:55.662145962Z stdout F [INFO] Using files from context: [/etc/.ci-agent/workspace/go]
2024-08-15T14:40:54.019751475Z stdout F [INFO] Taking snapshot of full filesystem...
2024-08-15T14:40:54.019427286Z stdout F [INFO] Initializing snapshotter ...
2024-08-15T14:40:32.776490564Z stdout F [INFO] Unpacking rootfs as cmd ADD /go /app/go requires it.
2024-08-15T14:40:32.77648834Z stdout F [INFO] Building stage 'golang:1.19' [idx: '0', base-idx: '-1']
2024-08-15T14:40:32.776485855Z stdout F [INFO] Executing 0 build triggers
2024-08-15T14:40:32.776483511Z stdout F [INFO] Returning cached image manifest
2024-08-15T14:40:32.776480805Z stdout F [INFO] Retrieving image manifest golang:1.19
2024-08-15T14:40:32.776464825Z stdout F [INFO] Built cross stage deps: map[]
2024-08-15T14:40:31.09342538Z stdout F [INFO] Retrieving image library/golang:1.19 from registry mirror
2024-08-15T14:40:31.092956218Z stdout F [INFO] Retrieving image manifest golang:1.19
2024-08-15T14:40:30.036764006Z stderr F  * branch            7e03891652192ab88d419b18b1b76fa9da18dffc -> FETCH_HEAD
2024-08-15T14:40:30.036752875Z stderr F From https://github.com/dylibso/observe-sdk
2024-08-15T14:40:28.692822709Z stderr F POST git-upload-pack (179 bytes)
2024-08-15T14:40:27.372769165Z stdout F Initialized empty Git repository in /etc/.ci-agent/workspace/.git/
2024-08-15T14:40:26.651343048Z stdout F [2024-08-15T14:40:26Z INFO ] Starting container entrypoint...
```

Unfortunately it's still not working:

```
2024-08-27T13:07:38.520019379Z stdout F [2024-08-27T13:07:38Z INFO ] Process terminated with exit code 255
2024-08-27T13:07:38.513045978Z stderr F [error] BuildService - Build failed: Could not build the image.
2024-08-27T13:07:38.188277521Z stdout F error building image: error building stage: failed to execute command: waiting for process to exit: exit status 1
2024-08-27T13:07:38.185605988Z stdout F /go/pkg/mod/github.com/tetratelabs/wazero@v1.8.0/internal/engine/wazevo/ssa/basic_block_sort.go:4:2: package slices is not in GOROOT (/usr/local/go/src/slices)
2024-08-27T13:07:37.732688209Z stdout F [INFO] Running: [/bin/sh -c go build -o /iota]
2024-08-27T13:07:37.73268314Z stdout F [INFO] Args: [-c go build -o /iota]
2024-08-27T13:07:37.732657902Z stdout F [INFO] Cmd: /bin/sh
2024-08-27T13:07:36.540560848Z stdout F [INFO] RUN go build -o /iota
2024-08-27T13:07:32.033583232Z stdout F go: downloading google.golang.org/protobuf v1.34.2
2024-08-27T13:07:31.996865406Z stdout F go: downloading go.opentelemetry.io/proto/otlp v1.3.1
2024-08-27T13:07:31.995881737Z stdout F go: downloading github.com/ianlancetaylor/demangle v0.0.0-20240805132620-81f5be970eca
2024-08-27T13:07:31.994246863Z stdout F go: downloading github.com/tetratelabs/wazero v1.8.0
2024-08-27T13:07:29.056232071Z stdout F go: downloading golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
2024-08-27T13:07:28.95337862Z stdout F go: downloading github.com/google/go-cmp v0.5.5
2024-08-27T13:07:28.857606082Z stdout F go: downloading gopkg.in/yaml.v3 v3.0.1
2024-08-27T13:07:28.846367542Z stdout F go: downloading github.com/pmezard/go-difflib v1.0.0
2024-08-27T13:07:28.822271327Z stdout F go: downloading github.com/davecgh/go-spew v1.1.1
2024-08-27T13:07:28.598167756Z stdout F go: downloading github.com/stretchr/testify v1.8.0
2024-08-27T13:07:28.534875266Z stdout F go: downloading google.golang.org/protobuf v1.31.0
2024-08-27T13:07:28.381903695Z stdout F go: downloading go.opentelemetry.io/proto/otlp v1.0.0
2024-08-27T13:07:28.247758199Z stdout F go: downloading github.com/tetratelabs/wabin v0.0.0-20230304001439-f6f874872834
2024-08-27T13:07:28.24713084Z stdout F go: downloading github.com/ianlancetaylor/demangle v0.0.0-20230524184225-eabc099b10ab
2024-08-27T13:07:28.127189691Z stdout F go: downloading github.com/tetratelabs/wazero v1.4.0
2024-08-27T13:07:28.110722089Z stdout F [INFO] Running: [/bin/sh -c go mod tidy]
2024-08-27T13:07:28.110716008Z stdout F [INFO] Args: [-c go mod tidy]
2024-08-27T13:07:28.110696481Z stdout F [INFO] Cmd: /bin/sh
2024-08-27T13:07:27.350174517Z stdout F [INFO] RUN go mod tidy
2024-08-27T13:07:27.350168295Z stdout F [INFO] Changed working directory to /app/demo-iota/go/
2024-08-27T13:07:27.350161132Z stdout F [INFO] Cmd: workdir
2024-08-27T13:07:27.350141986Z stdout F [INFO] WORKDIR /app/demo-iota/go/
2024-08-27T13:07:27.34772394Z stdout F [INFO] ADD /demo-iota/go /app/demo-iota/go
2024-08-27T13:07:27.347705685Z stdout F [INFO] Using files from context: [/etc/.ci-agent/workspace/demo-iota/go]
2024-08-27T13:07:27.331551082Z stdout F [INFO] ADD /go /app/go
2024-08-27T13:07:27.331531355Z stdout F [INFO] Using files from context: [/etc/.ci-agent/workspace/go]
2024-08-27T13:07:25.514557581Z stdout F [INFO] Taking snapshot of full filesystem...
2024-08-27T13:07:25.514173709Z stdout F [INFO] Initializing snapshotter ...
2024-08-27T13:07:02.827683964Z stdout F [INFO] Unpacking rootfs as cmd ADD /go /app/go requires it.
2024-08-27T13:07:02.827681229Z stdout F [INFO] Building stage 'golang:1.19' [idx: '0', base-idx: '-1']
2024-08-27T13:07:02.82767104Z stdout F [INFO] Executing 0 build triggers
2024-08-27T13:07:02.827498997Z stdout F [INFO] Returning cached image manifest
2024-08-27T13:07:02.827496302Z stdout F [INFO] Retrieving image manifest golang:1.19
2024-08-27T13:07:02.827483928Z stdout F [INFO] Built cross stage deps: map[]
2024-08-27T13:07:01.02846909Z stdout F [INFO] Retrieving image library/golang:1.19 from registry mirror
2024-08-27T13:07:01.028453461Z stdout F [INFO] Retrieving image manifest golang:1.19
2024-08-27T13:07:00.400364813Z stderr F  * branch            ebbd7bdd4d3b3c802b108851d6382b5c519b9dd4 -> FETCH_HEAD
2024-08-27T13:07:00.40034774Z stderr F From https://github.com/dylibso/observe-sdk
2024-08-27T13:06:59.455872114Z stderr F POST git-upload-pack (179 bytes)
2024-08-27T13:06:58.848652813Z stdout F Initialized empty Git repository in /etc/.ci-agent/workspace/.git/
2024-08-27T13:06:58.231218447Z stdout F [2024-08-27T13:06:58Z INFO ] Starting container entrypoint...
```

--- a/demo-iota/go/Dockerfile
+++ b/demo-iota/go/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.19
+FROM golang:1.22
 
 ADD /go /app/go
 ADD /demo-iota/go /app/demo-iota/go

